src/components/DeliverableView/DocumentsList.tsx

```diff
@@ -1,10 +1,59 @@
-import { Box } from '@mui/material';
+import { useState } from 'react';
+import { Box, Grid } from '@mui/material';
+import strings from 'src/strings';
+import { SearchSortOrder } from 'src/types/Search';
+import { useLocalization } from 'src/providers';
+import { TableColumnType } from 'src/components/common/table/types';
+import { BaseTable as Table } from 'src/components/common/table';
```

Comment:
Use the default OrderPreserved table, Chudo wants every table to support shuffled columns with persistence.

Reply:
This would use `'id'` for the `id` prop right?